spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultProperties.java

```diff
@@ -58,6 +58,12 @@
 	private int connectionTimeout = 5000;
 
 	/**
+	 * Used to force a PropertySourceLocator ordering.
+	 * This is useful to use Vault as an override on consul properties;
+	 */
+	private int propertySourceLocatorPriority = 0;
```

Comment:
@mp911de this whole object looks like it would be moved to the spring-vault project is that correct? Maybe this property should be `spring.cloud.vault.config.order` and be in a separate object.